site/src/main/webapp/WEB-INF/web.xml

```diff
@@ -108,6 +108,18 @@
   <servlet>
     <servlet-name>BinariesServlet</servlet-name>
     <servlet-class>org.hippoecm.hst.servlet.BinariesServlet</servlet-class>
+
+    <init-param>
+        <param-name>contentDispositionContentTypes</param-name>
+        <param-value>*/*</param-value>
```

Comment:
* Should PDFs open in new tab?
* What is max file size we should allow? Is this is in scope?
* What file types are allowed? Is this is in scope?
* Do we need to show file size / type?

Reply:
Questions forwarded to the product owner.

Reply:
And response now documented by BA in JIRA and implemented (allowed file types and max size configured, file sizes displayed; support for extra large files deferred to a separate, future story).